src/interface/read_config_file.py

```diff
@@ -0,0 +1,108 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, fileconfig): 
+            
+            
+            self.xmld = etree.parse(fileconfig)             
+            self.root = self.xmld.getroot()
+
+        
+
+    def get_output_file(self):
+        """ 
+        Returns the outputfile path, name and extension values from config file
+        listoutput: variable to assign all values read from config file
+        """
+        list_output = []
+        for output in self.root.findall('Output'):
+            path = output.find('Path').text
+            name = output.find('Name').text         
+            extension = output.find('Extension').text   
+
+
+        list_output.append(path)
+        list_output.append(name)
+        list_output.append(extension)
+
+        
+        return list_output
+
+    def list_of_file_types_supported(self):
+        """     
+        Returns a list with all files types values supported, it will be also get from config file
+        listtypes: variable of list type, it will have assigned all supported files, this info is read from config file
```

Comment:
the description mention that return a list of file types but the method does not have a return statement